### PR TITLE
OHRM-121 : In List and auto fills purge Employee Appear Bug 

### PR DESCRIPTION
**Pull request by AshanKooragoda**

### CODE REVIEW

File: symfony/plugins/orangehrmPimPlugin/modules/pim/actions/getEmployeeListAjaxAction.class.php [hunk 1]
```diff
@@ -23,41 +23,46 @@
...
-    	/**
-	 * get Red hat location by country
-	 * 
-	 */
-	public function execute($request){
+class getEmployeeListAjaxAction extends sfAction
+{
+
+    /**
+     * get Red hat location by country
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmPimPlugin/modules/pim/actions/getEmployeeListAjaxAction.class.php]: What this comment ?

File: symfony/plugins/orangehrmPimPlugin/modules/pim/actions/getEmployeeListAjaxAction.class.php [hunk 2]
```diff
@@ -23,41 +23,46 @@
...
 
         echo $jsonString;
         return sfView::NONE;
 
-	}
-    
-    protected function compareByName($employee1, $employee2) {
+    }
+
+    protected function compareByName($employee1, $employee2)
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmPimPlugin/modules/pim/actions/getEmployeeListAjaxAction.class.php]: no doc blocks

File: symfony/plugins/orangehrmPimPlugin/modules/pim/actions/getEmployeeListAjaxAction.class.php (hunk 1 above)
Comment (AshanKooragoda) [reply symfony/plugins/orangehrmPimPlugin/modules/pim/actions/getEmployeeListAjaxAction.class.php]: this is not a comment added by me

Comment (AshanKooragoda) [reply symfony/plugins/orangehrmPimPlugin/modules/pim/actions/getEmployeeListAjaxAction.class.php]: corrected

**Pull request closed without merging**